lib/inc/internal/facts/openbsd/networking_resolver.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file
+ * Declares the OpenBSD networking fact resolver.
+ */
+#pragma once
+
+#include "../bsd/networking_resolver.hpp"
+#include <map>
+#include <ifaddrs.h>
+
+namespace facter { namespace facts { namespace openbsd {
+
+    /**
+     * Responsible for resolving networking facts.
+     */
+    struct networking_resolver : bsd::networking_resolver
+    {
+     protected:
+        /**
+         * Gets the MTU of the link layer data.
+         * @param interface The name of the link layer interface.
+         * @param data The data pointer from the link layer interface.
+         * @return Returns The MTU of the interface.
+         */
+        virtual boost::optional<uint64_t> get_link_mtu(std::string const& interface, void* data) const;
```

Comment:
We should have an `override` declaration on this member, since it overrides the base member.